src/enumo/enumosym.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
```

Comment:
I think you can use egg's [`SymbolLang`](https://docs.rs/egg/latest/egg/struct.SymbolLang.html) instead of this module.